web/src/buddy_list.ts

```diff
@@ -78,7 +78,7 @@ type BuddyListRenderData = {
     total_human_subscribers_count: number;
     other_users_count: number;
     hide_headers: boolean;
-    all_participant_ids: Set<number>;
+    all_participant_ids: () => Set<number>;
```

Comment:
This should get renamed to like `get_all_participant_ids`; otherwise the name doesn't really hint the type.